src/cursor/cur_table.c

```diff
@@ -30,6 +30,13 @@ typedef struct {
     int (*f)(WT_CURSOR *);
 } WT_CURSOR_EXTRACTOR;
 
+/* Cursor bounds state. */
+typedef struct {
+    WT_ITEM lower_bound;
+    WT_ITEM upper_bound;
+    uint64_t flags;
+} WT_CURSOR_BOUNDS_STATE;
```

Comment:
Move to cursor.h

Reply:
[59a73d6](https://github.com/wiredtiger/wiredtiger/pull/8265/commits/59a73d6177d493f37cd1d9b9e8a5f5b0b9edb277)